src/js/chat.js

```diff
@@ -25,7 +25,9 @@ import './polyfills';
 import {CHAT_CHANNEL, CURRENT_USER} from './define-user';
 import '../css/chat.css';
 
-console.log(5);
+$("#disconnect").click(function () {
```

Comment:
This should go in the  `$( document ).ready` function (https://github.com/EteRNAgame/HTML-Chat/pull/48/files#diff-c481234f8b2a468d3f9c5823d7023653R301)